account_invoice_customer_no_autofollow/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Invoice partner no autofollow",
+    "summary": "Account invoice partner no autofollow",
```

Comment:
summary must contain some valuable summary. 
Eg "Do not add customer as follower in Invoices"

Reply:
Got it, I'll fix it